### ISSUE TITLE
Changes to indent rule for empty blocks in eslint@1

### ISSUE THREAD

**Issue by pdehaan** [eslint/eslint#3279]
Previously we had code similar to this dummy code below and it looks and linted fine in eslint@0.24.x. Since moving to eslint@1 (awesome work, btw!) it seems like we're getting a few rogue errors like:

> "Expected indentation of 4 characters but found 6"

---

### dummy.js
```js
/*eslint-env node */

var p = {};

var obj = {
  foo: function () {
    return new p()
      .then(function (ok) {
        return ok;
      }, function () {
        // ignore things
      });
  }
};
console.log(obj);
```

### .eslintrc
```json
{
  "extends": "eslint:recommended",
  "rules": {
    "handle-callback-err": 0,
    "no-console": 0,
    "strict": [2, "never"],
    "indent": [2, 2]
  }
}
```

### Output:
```
➜  fxa-esl1 git:(devdeps44) ✗ npm run lint

> fxa-content-server@0.43.0 lint /Users/pdehaan/dev/tmp/del/fxa-esl1
> eslint dummy.js


dummy.js
  12:8  error  Expected indentation of 4 characters but found 6  indent

✖ 1 problem (1 error, 0 warnings)
```

### Workaround:
If I'm understanding this correctly, this seems to be because my "ignore things" function is an empty block and doesn't have any statements. If I change that to `return;` then the `eslint dummy.js` succeeds.

**Comment by gyandeeps** [comment]
Multiline is a known issue.
Kinda related https://github.com/eslint/eslint/issues/1801

**Comment by nzakas** [comment]
@gyandeeps so is this a new bug? Or is it covered by another issue?
